.github/workflows/lint_python.yml

```diff
@@ -15,8 +15,8 @@ jobs:
       - run: bandit --recursive --skip B105,B110,B311,B605,B607 .
       - run: black --check .
       - run: codespell  # --ignore-words-list="" --skip=""
-      - run: flake8 --ignore=E12,E20,E226,E231,E265,E271,E40,E731,F401,W29,W605
-                    --max-complexity=10 --max-line-length=255 --show-source --statistics .
+      - run: flake8 --ignore=E203 --max-complexity=10 --max-line-length=255
```

Comment:
Why is the error E203 still ignored?
I read it is the whitespace before `:`
https://www.flake8rules.com/rules/E203.html

Reply:
See the URL in the commit message.